src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,16 @@
 // Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo: { completed, title, user } }) => {
+  const isCompleted = completed
+    ? `TodoInfo TodoInfo--completed`
+    : `TodoInfo`;
```

Comment:
Use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally